src/modules/uORB/Subscription.cpp

```diff
@@ -54,37 +54,99 @@
 #include "topics/vehicle_attitude_setpoint.h"
 #include "topics/vehicle_rates_setpoint.h"
 #include "topics/rc_channels.h"
+#include "topics/battery_status.h"
```

Comment:
The original thought was to only create C++ wrapper classes for the topics that were used, but this is something we can do in the future. I don't think it should be a priority in this PR.